### PR TITLE
Migration from cookie to localStorage, deleted ‘react-cookie’ pkg

### PR DESCRIPTION
**Pull request by joaquinTop**
Replacing cookies with local storage for session management.

### CODE REVIEW

File: src/actions/sessionActions.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import * as types from './actionTypes';
 import userClient from '../client/UsersServerClient';
 import targetClient from '../client/TargetsServerClient';
 import {resetTargets} from './targetActions';
-import cookie from 'react-cookie';
+import {setUser, removeUser} from '../utils/sessionHelper'
```
Comment (bernabe9) [inline src/actions/sessionActions.js]: leave a blank space between the brackets

File: src/containers/SignInPage.js [hunk 2]
```diff
@@ -5,13 +5,15 @@ import {bindActionCreators} from 'redux';
 import * as sessionActions from '../actions/sessionActions';
 import SignInForm from '../components/signin/SignInForm';
 import LandingRightSide from '../components/common/LandingRightSide';
-import cookie from 'react-cookie';
+import {getUser} from '../utils/sessionHelper'
```
Comment (bernabe9) [inline src/containers/SignInPage.js]: same

File: src/utils/sessionHelper.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+export const setUser = (user) => {
+  if (typeof(Storage) !== "undefined") {
+    localStorage.setItem('user', JSON.stringify(user));
+  }
+}
+
+export const getUser = () => {
+  let user;
```
Comment (bernabe9) [inline src/utils/sessionHelper.js]: could be a `const` instead of `let`

**Review by bernabe9 — CHANGES_REQUESTED** [review]

File: src/utils/sessionHelper.js (hunk 3 above)
Comment (joaquinTop) [reply src/utils/sessionHelper.js]: i can't use a const because the value is assigned after the if statement.

**Review by bernabe9 — APPROVED** [review]

**Pull request merged**